docs/roman/ramp_fitting/description.rst

```diff
@@ -193,6 +193,114 @@ Upon successful completion of this step, the status attribute ramp_fit will be s
+the resultants in the ramp.  By single-difference we mean the difference between two adjacent
+resultants in the ramp, while double-difference refers to the difference between a resultant
+and a resultant two steps away (the resultant adjacent to a resultant adjacent to the resultant
+in question).
+
+To compute these statistics, we need the single-difference excess slope :math:`\delta_{i, i+1}` and
+the double-difference excess slope :math:`\delta_{i, i+2}` are computed as:
+
+.. math::
+
+   \delta_{i, i+1} &= \frac{R_{i+1} - R_i} {\bar t_{i+1} - \bar t_i} - \hat \alpha
+
+   \delta_{i, i+2} &= \frac{R_{i+2} - R_i} {\bar t_{i+2} - \bar t_i} - \hat \alpha
+
+where :math:`\hat \alpha` is the slope computed by the ramp fitting algorithm. The
+variance in the excess slope:
+
+.. math::
+
+   Var(\delta_{i, j}) &= \frac {Var(R_j - R_i)} {(\bar t_j - \bar t_i)^2} + f_{corr}(\hat \alpha)
+
+   Var(R_j - R_i) &= \sigma_{RN} \left( \frac{1}{N_j} + \frac{1}{N_i} \right) + \hat \alpha \left[\tau_j + \tau_i - \min(\bar t_j, \bar t_i) \right]
+
+   f_{corr}(\hat \alpha) &= - \frac{\hat \alpha}{t_{M - 1} - t_0}
+
+where :math:`\sigma_{RN}` is the read noise. The single-difference statistic, :math:`s_i^\prime`,
+and double-difference statistic, :math:`s_i^{\prime\prime}` are,
+
+.. math::
+
+   s_i^\prime &= \frac{\delta_{i, i+1}} {\sqrt{Var(\delta_{i, i+1})}}
+
+   s_i^{\prime\prime} &= \frac{\delta_{i, i+2}} {\sqrt{Var(\delta_{i, i+2})}}.
+
+The statistic :math:`s_i` for each resultants :math:`0 \leq i \leq M - 1` (no differences from the last
+resultant are possible) is then computed as:
+
+.. math::
+   :nowrap:
+
+   \[
+   s_i =
+   \begin{cases}
+   s_i^\prime & \text{if } i = M - 2\\
+   \max(s_i^\prime, s_i^{\prime\prime}) & \text{otherwise}
+   \end{cases}
+   \]
+
+
+Finally, :math:`S = \max(s_i)` is the statistic used to determine if a jump has occurred in the fitted
+ramp. The possible jump resultants for this ramp are the resultants :math:`R_i` and :math:`R_{i+1}`,
+where :math:`i = \arg\max(s_i)`.
+
+Two possible jump resultants are necessary, because the statistics cannot determine which of the two
+adjacent resultants is the one affected by the jump. This is because if the jump occurs near the last
+read making up :math:`R_i`, then it might appear that :math:`R_{i+1}` has the jump, this jump will be
+picked up the :math:`s_i^{\prime\prime}` statistic. Using just the :math:`s_i^\prime` statistic, the
+jump would be incorrectly identified in :math:`R_{i+1}`.
+
+
+Threshold Function
+++++++++++++++++++
+
+Similarly to the statistic, the threshold function relies on the slope computed by the ramp fitting
+algorithm, :math:`\hat \alpha`. The function itself was determined empirically by running simulations
+of ramps with jumps and ramps without jumps. The threshold function was determined to be:
+
+.. math::
+
+   S_{\text{threshold}}(\hat \alpha) = 5.5 - \frac{1}{3}\log_{10}(\hat \alpha)
+
```

Comment:
Let's add a small amount of explanatory verbiage:
This corresponds to identifying jumps at 5.5 sigma when the count rate is 1 electron / s, 4.5 sigma when the count rate is 1000 electron / s, etc.  The decision was made to have the threshold depend on the count rate because pixels with lots of signal have larger uncertainties; this lets lower amplitude cosmics get identified in these cases.